cdec/installation-manager-tests/src/main/resources/bin/codenvy/audit/test-audit.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# CODENVY CONFIDENTIAL
+# ________________
+#
+# [2012] - [2015] Codenvy, S.A.
+# All Rights Reserved.
+# NOTICE: All information contained herein is, and remains
+# the property of Codenvy S.A. and its suppliers,
+# if any. The intellectual and technical concepts contained
+# herein are proprietary to Codenvy S.A.
+# and its suppliers and may be covered by U.S. and Foreign Patents,
+# patents in process, and are protected by trade secret or copyright law.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Codenvy S.A..
+#
+
+# load lib.sh from path stored in parameter 1
+. $1
+
+printAndLog "TEST CASE: Audit"
+vagrantUp ${SINGLE_NODE_VAGRANT_FILE}
+
+# install Codenvy on-prem
+installCodenvy ${LATEST_CODENVY_VERSION}
+validateInstalledCodenvyVersion ${LATEST_CODENVY_VERSION}
+
+# install IM CLI
+installImCliClient ${LATEST_IM_CLI_CLIENT_VERSION}
+validateInstalledImCliClientVersion ${LATEST_IM_CLI_CLIENT_VERSION}
+
+addCodenvyLicenseConfiguration
+storeCodenvyLicense
+
+authWithoutRealmAndServerDns "admin" "password"
+
+#create user "user1.im.test@email.com"
+doPost "application/json" "{\"name\":\"user1\",\"email\":\"user1.im.test@email.com\",\"password\":\"pwd123ABC\"}" "http://${HOST_URL}/api/user" "${TOKEN}"
+
+#create user "user2.im.test@email.com"
+doPost "application/json" "{\"name\":\"user2\",\"email\":\"user2.im.test@email.com\",\"password\":\"pwd123ABC\"}" "http://${HOST_URL}/api/user" "${TOKEN}"
+fetchJsonParameter "id"
+USER_ID=${OUTPUT}
+
+authWithoutRealmAndServerDns "user1" "pwd123ABC"
+# create workspace 1
+WORKSPACE_NAME="workspace-1"
+doPost "application/json" "{\"defaultEnv\":\"default\",\"commands\":[{\"commandLine\":\"mvn clean install -f $\{current.project.path}\",\"name\":\"build\",\"type\":\"mvn\",\"attributes\":{}}],\"projects\":[],\"name\":\"${WORKSPACE_NAME}\",\"environments\":{\"default\":{\"recipe\":{\"location\":\"codenvy/ubuntu_jdk8\",\"type\":\"dockerimage\"},\"machines\":{\"dev-machine\":{\"servers\":{},\"agents\":[\"org.eclipse.che.terminal\",\"org.eclipse.che.ws-agent\",\"org.eclipse.che.ssh\"],\"attributes\":{\"memoryLimitBytes\":1610612736},\"source\":{\"type\":\"dockerfile\",\"content\":\"FROM codenvy/ubuntu_jdk8\"}}}}},\"links\":[],\"description\":null}" "http://${HOST_URL}/api/workspace/?token=${TOKEN}"
+
+# create workspace 2
+WORKSPACE_NAME="workspace-2"
```

Comment:
From the readability point it's better to use separate constants for each workspace name and user name: WORKSPACE_NAME_1,  WORKSPACE_NAME_2,...; USER_1, USER_2, ....